API.md

```diff
@@ -348,7 +348,7 @@ If you use data functions rather than passing data on the `request` object, thes
     The provided function should perform whatever tasks are necessary to update your application's data.
     - `request` - the Hapi `request` object received by the handler. This will always be a POST request.
     - `data` - an object containing the data to be set, with a key for each named form component.
-    - returns - nothing, or a Joi validation result. If a Joi validation result is returned and it has an `error`
+    - returns - nothing, or an [error validation result object](#error-validation-result-object). The error validation result is returned and it has an `error`
```

Comment:
This _should_ be a Joi validation result at this point.

I think it might be the code that's wrong rather than the documentation!